File: mobile_flutter/lib/config/api_config.dart
/// API Configuration for the Flutter app
class ApiConfig {
  // Base URL for the backend API
  // Change this to your production URL when deploying
  static const String baseUrl = 'https://atiohaidar-backend.haidarprograming.workers.dev';
  
  // Alternative development URL (uncomment if needed)
  // static const String baseUrl = 'http://localhost:8787';
  
  // API version prefix
  static const String apiPrefix = '/api';
  
  // Full API URL
  static String get apiUrl => '$baseUrl$apiPrefix';
  
  // WebSocket URL for real-time chat
  static String get wsUrl => baseUrl.replaceFirst('https://', 'wss://').replaceFirst('http://', 'ws://');
  
  // Timeout settings
  static const Duration connectTimeout = Duration(seconds: 30);
  static const Duration receiveTimeout = Duration(seconds: 30);
}

File: android/app/src/main/java/com/example/portoflio_android/data/network/ApiConfig.kt
package com.example.portoflio_android.data.network

/**
 * API configuration constants.
 */
object ApiConfig {
    // Use 10.0.2.2 for Android Emulator to access localhost
    const val BASE_URL_DEV = "https://backend.atiohaidar.workers.dev"
//    const val BASE_URL_DEV = "http://10.0.2.2:8787"

    const val BASE_URL_PROD = "https://backend.atiohaidar.workers.dev"
    
    // Toggle this for development vs production
    const val IS_PRODUCTION = false
    
    val BASE_URL: String
        get() = if (IS_PRODUCTION) BASE_URL_PROD else BASE_URL_DEV
}
